src/bindings/python/tests_compatibility/test_ngraph/test_ops_binary.py

```diff
@@ -7,203 +7,191 @@
+        (ng.greater_equal, np.greater_equal, Type.boolean),
+        (ng.less, np.less, Type.boolean),
+        (ng.less_equal, np.less_equal, Type.boolean),
     ],
 )
-def test_binary_op(ng_api_helper, numpy_function):
-    runtime = get_runtime()
-
+def test_binary_op(ng_api_helper, numpy_function, expected_type):
     shape = [2, 2]
     parameter_a = ng.parameter(shape, name="A", dtype=np.float32)
     parameter_b = ng.parameter(shape, name="B", dtype=np.float32)
 
     model = ng_api_helper(parameter_a, parameter_b)
-    computation = runtime.computation(model, parameter_a, parameter_b)
 
     value_a = np.array([[1, 2], [3, 4]], dtype=np.float32)
     value_b = np.array([[5, 6], [7, 8]], dtype=np.float32)
 
-    result = computation(value_a, value_b)
-    expected = numpy_function(value_a, value_b)
-    assert np.allclose(result, expected)
+    expected_shape = numpy_function(value_a, value_b).shape
+    assert model.get_output_size() == 1
+    assert list(model.get_output_shape(0)) == list(expected_shape)
+    assert model.get_output_element_type(0) == expected_type
 
 
 @pytest.mark.parametrize(
-    "ng_api_helper,numpy_function",
+    ("ng_api_helper", "numpy_function", "expected_type"),
     [
-        (ng.add, np.add),
-        (ng.divide, np.divide),
-        (ng.multiply, np.multiply),
-        (ng.subtract, np.subtract),
-        (ng.minimum, np.minimum),
-        (ng.maximum, np.maximum),
-        (ng.mod, np.mod),
-        (ng.equal, np.equal),
-        (ng.not_equal, np.not_equal),
-        (ng.greater, np.greater),
-        (ng.greater_equal, np.greater_equal),
-        (ng.less, np.less),
-        (ng.less_equal, np.less_equal),
+        (ng.add, np.add, Type.f32),
+        (ng.divide, np.divide, Type.f32),
+        (ng.multiply, np.multiply, Type.f32),
+        (ng.subtract, np.subtract, Type.f32),
+        (ng.minimum, np.minimum, Type.f32),
+        (ng.maximum, np.maximum, Type.f32),
+        (ng.mod, np.mod, Type.f32),
+        (ng.equal, np.equal, Type.boolean),
+        (ng.not_equal, np.not_equal, Type.boolean),
+        (ng.greater, np.greater, Type.boolean),
+        (ng.greater_equal, np.greater_equal, Type.boolean),
+        (ng.less, np.less, Type.boolean),
+        (ng.less_equal, np.less_equal, Type.boolean),
     ],
 )
-def test_binary_op_with_scalar(ng_api_helper, numpy_function):
-    runtime = get_runtime()
-
+def test_binary_op_with_scalar(ng_api_helper, numpy_function, expected_type):
     value_a = np.array([[1, 2], [3, 4]], dtype=np.float32)
     value_b = np.array([[5, 6], [7, 8]], dtype=np.float32)
 
     shape = [2, 2]
     parameter_a = ng.parameter(shape, name="A", dtype=np.float32)
 
     model = ng_api_helper(parameter_a, value_b)
-    computation = runtime.computation(model, parameter_a)
-
-    result = computation(value_a)
-    expected = numpy_function(value_a, value_b)
-    assert np.allclose(result, expected)
+    expected_shape = numpy_function(value_a, value_b).shape
```

Comment:
There is no need to call the numpy function just for the output shape calculation, especially if the shape doesn't change between operations. 
The expected shape will be the same broadcasted shape each time.

Reply:
Good catch, I removed the numpy calls.